### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse – New Feature_Edit Configuration popup_Keyboard: Focus is going to “Close” button of "Edit configuration popup" but it is not visible to the user.

### ISSUE THREAD

**Issue by v-lasiva** [microsoft/azure-tools-for-java#3593]
[Check out Accessibility Insights!](url)- Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–New Feature; #MAS2.4.7; #Keyboard; #C+E-H; #JavatoolingforIntelliJ/Eclipse;

**Environment Details:**
URL: https://www.jetbrains.com/idea/download/#section=windows
Version: 2019.2.2
Build: 192.6603.28
MAS Violated: MAS2.4.7

**NOTE:**
**Same issue is observed in “Create Spark Scala” Scenario also.
(New-->Project and press enter
Select Azure Spark/HDInsight > Spark Project with Samples (Scala) and press Next.)
Here also focus is not visible on “Close” button.**

**Same issue is observed in "Link Cluster-Read only-Job Access Denied", "Link Cluster-Read only-Storage Access Denied", "Link Cluster-Read only-Storage Access Denied for Linked Clusters" scenarios also.**

**Repro Steps:**
1.Download IntelliJ application from the URL:  https://www.jetbrains.com/idea/download/#section=windows .
2.Launch IntelliJ IDEA.
3.Tab till Create New Project button and press enter
4.Tab till Azure Spark/HDInsight-->Spark Project Scala
5.Tab till Build Tools-->Maven
6.Tab till Next button and press enter
7.Tab till finish button and press enter
8.Navigate to project name-->src-->main-->scala-->project name and open the context menu using keyboard key
9.Tab till Submit Spark Application and press enter
10.Verify whether focus is visible on “Close” button of "Edit configuration popup" section or not.

**Actual Result:**
Focus is going to “Close” button of "Edit configuration popup", but it is not visible to the user.

**Expected Result:**
When focus is going to “Close” button of "Edit configuration popup", Keyboard focus must be visible to the user.

**User Impact:**
Keyboard users will face difficulty in knowing where the focus is pointing if focus is not visible on “close” button.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

[Bug_1_Focus is not visible on close button.pptx](https://github.com/microsoft/azure-tools-for-java/files/3641734/Bug_1_Focus.is.not.visible.on.close.button.pptx)

**Comment by t-rufang** [comment]
Note:
![image](https://user-images.githubusercontent.com/32627233/65597569-057e5a80-dfcc-11e9-8db7-aefe1c67460f.png)

Tab three times and the focus is in close button, but it's not visible.

**Comment by t-rufang** [comment]
The run configuration dialog is controlled by Intellij. If we create an `Application` run configuration, we can repro the same issue.
Check the gif below, after we create an `Application` configuration and tab till the focus at `Apply` button. Tab again and the focus is invisible but if we click enter, we can find the dialog is closed. Therefore, we can infer that the focus was at `close` button but it's invisible.
Therefore, this issue should be owned by IntelliJ team and it's out of the scope of our toolkit.

![issue_3593](https://user-images.githubusercontent.com/32627233/66986763-136a5b80-f0f2-11e9-9c13-acd114f43531.gif)

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749696
